Resources/Prototypes/Floof/Entities/Mobs/NPCs/xenopet.yml

```diff
@@ -0,0 +1,308 @@
+- type: entity
+  name: basic
+  id: MobSpaceFriend
+  parent: MobXenoRounyNPC
+  description: It looks friendly. Why don't you give it a hug?
+  components:
+  - type: NpcFactionMember
+    factions:
+    - PetsNT
+  - type: MovementAlwaysTouching
+  - type: MeleeWeapon
+    soundHit:
+        path: /Audio/Weapons/Xeno/alien_claw_flesh3.ogg
+    angle: 0
+    animation: WeaponArcClaw
+    damage:
+      types:
+        Blunt: 5
+        Slash: 5
+  - type: Puller
+    needsHands: false
+  - type: GhostTakeoverAvailable
+
+- type: entity
+  name: Neutral Rouny
+  id: MobXenoNeutralRouny
+  parent: MobSpaceFriend
+  description: They mostly come at night. Mostly.
+  components:
+  - type: NpcFactionMember
+    factions:
+    - Passive
+  - type: LanguageKnowledge
+    speaks:
+    - Xeno
+    understands:
+    - Xeno
+    - GalacticCommon
+
+- type: entity
+  name: Neutral Praetorian
+  id: MobXenoNeutralPraetorian
+  parent: MobXenoPraetorianNPC
+  description: They mostly come at night. Mostly.
+  components:
+  - type: NpcFactionMember
+    factions:
+    - Passive
+  - type: Puller
+    needsHands: false
+  - type: LanguageKnowledge
+    speaks:
+    - Xeno
+    understands:
+    - Xeno
+    - GalacticCommon
+
+- type: entity
+  name: Neutral Drone
+  id: MobXenoNeutralDrone
+  parent: MobXenoDroneNPC
+  description: They mostly come at night. Mostly.
+  components:
+  - type: LanguageKnowledge
+    speaks:
+    - Xeno
+    understands:
+    - Xeno
+    - GalacticCommon

```

Comment:
If neureal, is this one missing a faction?

Reply:
thanks for catching this, I didn't realize I had forgotten to give this one the passive faction.